airflow/providers/hashicorp/_internal_client/vault_client.py

```diff
@@ -472,6 +472,6 @@ def create_or_update_secret(
             )
         else:
             response = self.client.secrets.kv.v2.create_or_update_secret(
-                secret_path=secret_path, secret=secret, mount_point=mount_point, cas=cas
+                path=secret_path, secret=secret, mount_point=mount_point, cas=cas
```

Comment:
I think we have the same problem for version 1, could you fix it?

Reply:
Sure, I've just updated the v1 as well.